ios-trivia-game/ViewControllers/QuestionViewController.swift

```diff
@@ -50,12 +86,78 @@ class QuestionViewController: UIViewController {
         if (nav?.topViewController is AnswerViewController) {
             let destination = nav?.topViewController as! AnswerViewController
             destination.question = question
-            destination.roomId = roomId
+            destination.roomId = self.roomId
+
+            
+            print ("Preparing for segue: \(self.roomId)")
+            // @Nari or @Zhia: Feel free to remove the hardcoded values when you get to this
+            destination.roomId = (self.roomId)!
         }
     }
+    
+    @IBAction func quitGame(_ sender: Any) {
+        countdownTimer.invalidate()
+        Utilities.quitGame(controller: self)
+    }
 
     @IBAction func onAnswerSubmit(_ sender: UIButton) {
         // Temporary to demonstrate AnswerViewController
         // @Nari or @Zhia: Feel free to remove the hardcoded values when you get to this
+        let answerText = self.answerField.text
+        
```

Comment:
I don't think calling invalidate() here is desired because that means the user would be brought to the answer view right away. The flow in my mind is: 

Submit your answer -> Wait until time runs out -> See the list of answers from everyone -> Pick your correct answer

If we don't, then step 3 in the list of answers would only contain answers from whoever submitted first.